rmf_site_format/src/sdf.rs

```diff
@@ -49,6 +50,8 @@ pub enum SdfConversionError {
     BrokenModelInstanceReference(u32),
     #[error("Entity [{0}] referenced a non existing model description")]
     BrokenModelDescriptionReference(u32),
+    #[error("Failed deserializing world template")]
+    CorruptedWorldTemplate,
```

Comment:
Let's make this

```rust
CorruptedWorldTemplate(String)
```

and put the `Err(String)` from `WORLD_TEMPLATE` into it when an error occurs. You should also change the error message to something like

```rust
#[error("Failed to deserialize world template: {0}")]
```

so the details of the error will be included in the `SdfConversionError` message.